python/tests/unit/arcticdb/version_store/test_update.py

```diff
@@ -468,3 +468,34 @@ def test_update_pickled_data(lmdb_version_store):
     df2 = pd.DataFrame({"a": [1000]}, index=idx[1:2])
     with pytest.raises(InternalException) as e_info:
         lmdb_version_store.update(symbol, df2)
+
+
+def test_update_with_daterange_different_column_group_size(arcticdb_test_lmdb_config, lib_name):
+    local_lib_cfg = arcticdb_test_lmdb_config(lib_name)
```

Comment:
If you create a library by hand from config, then you must clear it up. Otherwise, it will waste disk space in Windows builds.

To avoid this complexity, the `version_store_factory` has an argument `reuse_name` which you can set to `True` in order to recreate another library instance sharing the storage of an existing (i.e. also pass `name=...` to the `factory`) instance but with different configuration.